WasatchNET/Spectrometer.cs

```diff
@@ -2327,7 +2436,7 @@ protected virtual double[] getSpectrumRaw(bool skipTrigger=false)
             if (triggerSource_ == TRIGGER_SOURCE.INTERNAL && autoTrigger && !skipTrigger)
                 sendSWTrigger();
 
-            if (isStroker)
+            if (true || isStroker)

```

Comment:
Are you planning on changing this back? Do you know if this is the only place in the getspectrum call stack with a delay? (I could go check)

Reply:
I think I'm planning on leaving it for now.  I'm of mixed feelings about this, and always have been.  Conceptually, I like the idea of beginning the blocking wait right after sending the ACQUIRE, such that the read timeout will need to at least span the actual integration time.  However, while checking one customer's logfiles, I've seen cases where the getSpectrumRaw() is completing within 20ms of sending the ACQUIRE, *even though* the integration time was 500ms.  In other words, it was CLEARLY reading spectral data (or fragments thereof) from a previous ACQUIRE, as the last-sent ACQUIRE could not possibly have completed by then.  Enforcing the wait seems a simple way of avoiding some obvious opportunities for timing errors, by maximizing the probability that the spectrum we read from the bulk endpoint will actually be the right one.

I can confirm that this is the only Thread.Sleep() in WasatchNET.Spectrometer which is based on integration time; the others are all small or fixed delays.